docs/index.rst

```diff
@@ -34,21 +64,22 @@ Learn more about contributing to this repository in our :doc:`Contribution Guide
 
    Home page <self>
    Installation <installation>
-   Getting Started <tutorials/quickstart>
-   Tutorials <tutorials>
```

Comment:
We had collapsed Tutorials and Example Notebooks into a single section intentionally. What's the reason to split them back out?